go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/asecurityteam/runhttp
```

Comment:
just FYI this go-mod file was so old it was doing the legacy style of keeping all the modules in one `require` section. Once we got to versions 1.16+ go started breaking up the go.mod file to add a separate section for the `indirect` dependencies.